azcli_conformity.sh

```diff
@@ -1,5 +1,18 @@
-# Define Variables for the Configurations
+#!/bin/bash
+
+# Fail the script in case any of the commands get an error
+set -e 
+
+# Define Variables for the Configurations using uuidgen or request an Application Secret if you are using Azure Cloud Shell
 export APP_SECRET=$(uuidgen)
+if [ -z $APP_SECRET ]
```

Comment:
❓ In what scenario would it be possible for `$APP_SECRET` not have a value?

Reply:
Because uuidgen is not available on Cloud Shell. That is why we create the manual process for the customer to insert the App Secret value